management/src/main/java/com/bitwisekaizen/sdss/dto/IscsiTargetSpec.java

```diff
@@ -0,0 +1,49 @@
+package com.bitwisekaizen.sdss.dto;
+
+import java.util.List;
+
+/**
+ * Spec that describes the ISCSI target
+ */
+public class IscsiTargetSpec {
```

Comment:
I think I might like to remain consistent with naming here.  IscsiTargetRequest would be consistent with PxeSessionRequest in pxeasy.  Spec is pretty general and can mean either input or output.